### PR TITLE
Edit profile

### PR DESCRIPTION
**Pull request by luckeyodeh**

### CODE REVIEW

File: src/components/edit-profile.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                type="text"
+                className="edit-profile-input"
+                placeholder="First Name" 
+              />
+              <input
+                type="text"
+                className="edit-profile-input" 
+                placeholder="Last Name" 
+              />
+              <input
```
Comment (jihdeh) [inline src/components/edit-profile.js]: Let's remove phone number field, email field, those ones won't change, let's add state and country.

File: src/styles/searchpage.css [hunk 2]
```diff
@@ -5,7 +5,7 @@
   padding-top: 9em;
 }
 
-#search-input-form {
+/*#search-input-form {
```
Comment (jihdeh) [inline src/styles/searchpage.css]: delete this if we aren't using it

File: src/styles/searchpage.css [hunk 3]
```diff
@@ -130,13 +126,13 @@
     margin-top: -8%;
     margin-bottom: 7%;
   }*/
-  .contestant-search__input {
+  /*.contestant-search__input {
```
Comment (jihdeh) [inline src/styles/searchpage.css]: delete this too

**Comment by jihdeh** [comment]
Few comments everyother thing looks good

File: src/styles/searchpage.css (hunk 3 above)
Comment (luckeyodeh) [reply src/styles/searchpage.css]: ok

File: src/styles/searchpage.css (hunk 2 above)
Comment (luckeyodeh) [reply src/styles/searchpage.css]: ok

File: src/components/edit-profile.js (hunk 1 above)
Comment (luckeyodeh) [reply src/components/edit-profile.js]: ok

**Pull request merged**